upcase_booklist_app/practice.py

```diff
@@ -0,0 +1,29 @@
+from flask import Flask, render_template, url_for
+app = Flask(__name__)
+
+
+single_book_details = {'title': 'Python Crash Course', 'author': "Eric Matthes", 'summary': 'A hands-on introduction to programming with Python.', 'date': '2016', 'image': 'img/python_crash_course.jpeg'}
```

Comment:
Recommend running [Black](https://github.com/python/black) to make your styles consistent.

But good use of a dictionary for your data store.